releases/20.0.0.3/kernel/Dockerfile.ubi.ibmjava8

```diff
@@ -14,8 +14,8 @@ LABEL org.opencontainers.image.authors="Arthur De Magalhaes, Chris Potter" \
       vendor="Open Liberty" \
       name="Open Liberty" \
       version="$LIBERTY_VERSION" \
-      summary="Image for Open Liberty with IBM's SFJ and UBI minimal" \
-      description="This image contains the Open Liberty runtime with IBM's SFJ and Red Hat UBI minimal as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image"
+      summary="Image for Open Liberty with IBM's SFJ and UBI 8" \
```

Comment:
This line and the one below still say SFJ.

Reply:
Ah alright, wasn't sure if that needed updating or not. I'll add those changes in 👍